.gitignore

```diff
@@ -2,3 +2,5 @@
 *.so
 *.o
 Dockerfile
+ptrack--2.1.sql
```

Comment:
`ptrack--2.1.sql` is a base extension sql file, it should not be in the `.gitignore`. Check `Makefile`, I have changed things a bit
```suggestion
```

Reply:
Ok, but now I need to find and recheck an extension which contains a similar sql in its .gitignore file.